src/ip/Address.cc

```diff
@@ -945,7 +945,7 @@ Ip::Address::getSockAddr(struct sockaddr_storage &addr, const int family) const
 
     if ( family == AF_INET && !isIPv4()) {
         // FIXME INET6: caller using the wrong socket type!
-        debugs(14, DBG_CRITICAL, HERE << "Ip::Address::getSockAddr : Cannot convert non-IPv4 to IPv4. from " << *this);
+        debugs(14, DBG_CRITICAL, "Ip::Address::getSockAddr : Cannot convert non-IPv4 to IPv4. from " << *this);
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required.